src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -329,6 +333,23 @@ public String getInvalidityInfoForGoogleId(String googleId) {
     }
 
     /**
+     * Checks if other instructors also set visibility false.
+     * @return An explanation of why instructor can't set visibility to false.
+     *         Returns an empty string if the instructor can set visibility to false.
+     */
+    public String getInvalidityInfoForisDisplayedToStudents(String courseId, String email) {
+
+        Logic logic = new Logic();
+        List<InstructorAttributes> instructors = logic.getInstructorsForCourse(courseId);
+        for (InstructorAttributes instructor : instructors) {
+            if (instructor.isDisplayedToStudents && !instructor.email.equals(email)) {
+                return "";
+            }
+        }
+        return IS_DISPLAYED_TO_STUDENTS_ERROR;
+    }
+
+    /**
```

Comment:
Sometimes, it's good to get an early review, otherwise one might have worked too much on the wrong solution. @mukesh14149 did the right thing here.